src/Transactions.sol

```diff
@@ -226,6 +287,9 @@ library Transactions {
         return txStruct;
     }
 
+    /// @notice decode a EIP-1559 transaction from RLP.
+    /// @param rlp is the encoded RLP bytes.
+    /// @return txStruct the transaction structure.
```

Comment:
Would it be possible to replace `txStruct` with `EIP1559 struct` here?

Reply:
Mm.. not really, that first word is the name of the return argument and I am not sure I can use EIP1559 as return name.